src/guide/1/geofencing/javascript/snippets/subscribe.js

```diff
@@ -0,0 +1,15 @@
+const filter = {exists: {field: 'message'}};
+const callback = (notification) => {
```

Comment:
(optional / nitpicking) Parenthesis are optionals here:

```suggestion
const callback = notification => {
```